vue-frontend/src/components/contact/ContactFormFooter.vue

```diff
@@ -12,9 +12,12 @@
               required
               autocomplete="given-username"
               v-model="name"
-              :disabled="disableInput"
               placeholder=" "
-              @input="showNameValidationError = name.trim().length === 0"
+              @input="
+                showNameValidationError =
+                  (name.trim().length === 0 && name.length > 0) ||
+                  (name.trim().length === 0 && showNameValidationError)
```

Comment:
Need to use an `event` in place of the `name` instead of all these conditions. Vue3 is working in that way only